source/MaterialXRuntime/RtNodeGraph.cpp

```diff
@@ -15,8 +16,7 @@ namespace MaterialX
 
 namespace
 {
-    static const RtToken SOCKETS("_nodegraph_internal_sockets");
-    static const RtToken NODEGRAPH1("nodegraph1");
+    const RtToken SOCKETS("_nodegraph_internal_sockets");
```

Comment:
You don't need the static keyword any more?

Reply:
Since it's in an anonymous namespace it has internal linkage already, so static is not needed. For a file global variable like this I think you can use either the static keyword _or_ an anonymous namespace to achieve internal linkage. Using both does compile fine but it should not be needed. 

Here's a discussion about why some think anonymous namespace is better in the general case:
https://stackoverflow.com/questions/4977252/why-an-unnamed-namespace-is-a-superior-alternative-to-static